src/api/participantsRouter.ts

```diff
@@ -253,7 +253,15 @@ export function createParticipantsRouter() {
         ...users.map((user) =>
           assignClientRoleToUser(kcAdminClient, user.email, 'api-participant-member')
         ),
-        participant!.$query().upsertGraph(data),
+        Participant.query().upsertGraph(
+          {
+            id: participant!.id!,
+            ...data,
+          },
+          {
+            relate: true,
+          }
+        ),
```

Comment:
Do we also need to update the audit trail here too?

Reply:
You are right, I forgot this requirement. That will involves db migration, I will create a separate branch for that

Reply:
cool!